### PR TITLE
Clean up imports in bodhi.server.models.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

**Comment by cverna** [comment]
Nice, clean and ordered 🥇 
LGTM

File: bodhi/server/models.py [hunk 1]
```diff
@@ -14,60 +14,49 @@
...
-import time
-import uuid
-
-from textwrap import wrap
-from datetime import datetime
 from collections import defaultdict
-
+from datetime import datetime
+from textwrap import wrap
 try:
```
Comment (jeremycline) [inline bodhi/server/models.py]: You _could_ use six to make this import less ugly, but it's up to you.

**Review by jeremycline — APPROVED** [review]
Looks tidy. There's one totally optional note :shipit:

File: bodhi/server/models.py (hunk 1 above)
Comment (bowlofeggs) [reply bodhi/server/models.py]: Cool, I'll use six. Thanks!

**Pull request merged**